actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,7 +45,11 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller = controller.to_s =~ %r{\arails/(info|mailers|welcome)}
+        internal_asset = Rails.application.config.respond_to?(:assets) &&
```

Comment:
Thing is sprockets is not included when using Rails API. Anyway maybe we can fix this in sprockets railtie but unsure what's better given the current way that sprockets is hooking into AV.